domain/src/main/java/gov/cms/ab2d/domain/Role.java

```diff
@@ -0,0 +1,33 @@
+package gov.cms.ab2d.domain;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.NaturalId;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Getter
+@Setter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+public class Role {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @NaturalId
+    @EqualsAndHashCode.Include
+    @Column(nullable = false, unique = true)
+    private String name;
+
+    @ManyToMany(mappedBy = "roles")
```

Comment:
I don't anticipate us needing to ever navigate from Role to Users based on the requirements. We can remove this.

Reply:
Removed the bi-directional association back to Set<User> entity.